perun/utils/structs/check_public.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
```

Comment:
`_public` sounds little bit weird. Maybe just call `check_structs`, etc. since we have `common_structs` and hence it will be uniform?

Reply:
Renamed.